packages/reactive-element/src/css-tag.ts

```diff
@@ -103,10 +105,12 @@ type ConstructableCSSResult = CSSResult & {
   ): CSSResult;
 };
 
-const textFromCSSResult = (value: CSSResultGroup | number) => {
+const textFromCSSResult = (value: CSSResultGroup | StaticValue | number) => {
   // This property needs to remain unminified.
   if ((value as CSSResult)['_$cssResult$'] === true) {
     return (value as CSSResult).cssText;
+  } else if ((value as StaticValue)['_$litStatic$']) {
+    return (value as StaticValue)['_$litStatic$'];
```

Comment:
Agree that we should use the same check as in unwrapStatic, and it'd be fine to export that function.

The brand is `Symbol.for('')`, so that it works with multiple versions of Lit in the page. There's no issues with exporting it, but probably better to instead export unwrapStaticValue.

We use a symbol for the brand because JSON.parse won't produce it (so malicious JSON can't impersonate a static value), multiple versions of Lit on the page can all have the same brand value, and it's quick to compare against.

Reply:
Would it make sense to export an `isStaticValue(value: unknown)` too in that case? I should reduce the footprint slightly since it can be minified and would compensate the additional bytes the `import` would add.

Reply:
After thinking about this a bit more I don't think that this is the correct wat to go about this change. It's much more natural to add a new `css` to `lit-html/static.js`. 

It would be more inline with how user would consume the static family of packages. Additionally it would not alter the size of the core package. 

I guess that might be a substantial enough change to warrant a separate discussion though?

Reply:
I don't think a `css` tag needs to live in the `lit-html` package as it has no use on its own and only makes sense to use in the `static styles` field of a reactive element.

The main reason to consider this change was that `literal` tag has basically the same security guarantee as `css` in that it's considered developer-authored string, so a `css` tag could allow `literal` tagged strings as values, just like it accepts other `css` tagged values. For that logic, the original approach here of widening the accepted values makes more sense to me.

> Would it make sense to export an isStaticValue(value: unknown) too in that case? It should reduce the footprint slightly since it can be minified and would compensate the additional bytes the import would add.

This adds more bytes to `lit-html` instead and won't get much minification anyway since `['_$litStatic$']` check needs to stay un-minified.

If we still want to do this, I think it should look something like
```suggestion
  } else if ((value as StaticValue)['_$litStatic$'] !== undefined) {
    return unwrapStaticValue(value as StaticValue) ?? '';
```

But I have reservations about adding this seeing it needs a new cross dependency over just doing the workaround proposed in the original issue.

Reply:
Sorry for the long wait. I have updated the PR with the requested changes. Let feel free to close this PR and the associated issue if this change is not desired. 

I am leaning towards this not being worth what it cost in extra bytes, it is not _that_ useful, merely a convenience.

Reply:
Thank you. The team had a discussion about this just now and do feel the effort to make this work doesn't justify the benefit. We really appreciate you giving it a try though.

Reply:
Indeed, thanks for trying this @filimon-danopoulos! I feel like something like this should work and we should keep our minds open to another approach or refactor that enables this change.

Reply:
No worries, was fun dipping my toes! :)